src/ui/pages/HomePage.js

```diff
@@ -5,6 +5,42 @@ export class HomePage {
     this.page = page;
     this.yourFeedTab = page.getByText('Your Feed');
     this.newArticleLink = page.getByRole('link', { name: 'New Article' });
+    this.globalFeedTab = page.getByText('Global Feed');
+    this.articlePreview = page.locator('.article-preview');
+    this.articleTitle = this.articlePreview.locator('h1');
+    this.yourFeedTab = page.getByText('Your Feed');
+  }
+
+  async assertTabisEmpty() {
```

Comment:
```suggestion
  async assertTabIsEmpty() {
```

or `assertNoArticlesMessageIsVisible`